open-api/immich-openapi-specs.json

```diff
@@ -4337,14 +4337,16 @@
     },
     "/server-info": {
       "get": {
+        "deprecated": true,
+        "description": "This property was deprecated in v1.106.0",
```

Comment:
oh this might be a little bug that it should say endpoint instead of property. looks like it uses the same description for both.

Reply:
Yup, fixing this is actually not super-trivial. I think I'd prefer a neutral name that works for both properties and endpoints